core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -652,4 +695,19 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest with Dou
       f.cramersV.isEmpty shouldBe true
     }
   }
+
+  it should "correctly return the descaled coefficient for linear regression, " +
+    "when standardization is on" in {
+
+    // Since 5000 & 1 are always returned as the coefficients of the model
+    // trained on unstandardized data and we can analytically calculate
+    // the scaled version of them by the linear regression formula, the coefficients
+    // of the model trained on standardized data should be within a small distance of the analytical formula.
+
+    // difference between the real coefficient and the analytical formula
+    val absError = math.abs(orginalbigCoeff * math.sqrt(smallFeatureVariance) / labelStd - descaledbigCoeff)
+    val absError2 = math.abs(originalsmallCoeff * math.sqrt(bigFeatureVariance) / labelStd - descaledsmallCoeff)
+    absError < 0.01 shouldBe true
+    absError2 < 0.01 shouldBe true
+  }
```

Comment:
please add a test for logistic regression as well

Reply:
I just checked again and I couldn't find an example where the feature contributions are on wildly different scales for logistic regression, so i will remove that case from the descaling logic, no test needed for logistic regression case.